src/validators/bartio/defaultValidatorList.json

```diff
@@ -774,6 +774,14 @@
       "description": "Professional staking service. 7/24 monitoring and best uptime. Huginn is an organization that aims to teach its community about Cosmos SDK and Blockchain.",
       "website": "https://huginn.tech/",
       "twitter": "https://x.com/HuginnStake"
+    },
+    {
+      "id": "0x35DaAEB5B592d963D6B3b5f2f7EB71e259C96f98",
+      "logoURI": "https://gumi-logo.s3.ap-northeast-1.amazonaws.com/images.png",
```

Comment:
Please prefix your logoURI with cloudinary url and upload the asset to `src/assets`

Reply:
@meithebear is there a way to to the modification by yourself here, @gumi-node made a Github issue https://github.com/berachain/default-lists/issues/154 like 4 days ago.

I can ping them on Discord but it's in the middle of the night in Japan right now.

Reply:
yes but i need the asset to be uploaded first.